airflow/dags/util.py

```diff
@@ -100,14 +102,34 @@ def service_request(url: str, data: dict):
     try:
         resp = requests.post(url, json=data, headers=receiving_service_headers)
         resp.raise_for_status()
+        kwargs['ti'].xcom_push(key='response_status', value=resp.status_code)
```

Comment:
Add comment that you are saving the response status for checking later.

What will happen if a step runs twice in a row? Will it save both values?

Reply:
In that scenario I believe the only the latest value will be saved for retrieval. If this becomes a problem I think we can use a combination of xcom keys and task ids to differentiate multiple calls.

https://airflow.apache.org/docs/apache-airflow/stable/concepts.html?highlight=xcom#xcoms